util/types/src/prelude.rs

```diff
@@ -15,7 +18,41 @@ impl<T> ShouldBeOk<T> for Option<T> {
 // Use for verify
 impl<T> ShouldBeOk<T> for molecule::error::VerificationResult<T> {
     fn should_be_ok(self) -> T {
-        self.unwrap_or_else(|err| panic!("verify slice should be ok, but {:?}", err))
+        self.unwrap_or_else(|err| panic!("verify slice should be ok, but {}", err))
+    }
+}
+
+pub trait FromSliceShouldBeOk<'r>: Reader<'r> {
+    fn from_slice_should_be_ok(slice: &'r [u8]) -> Self;
+}
+
+impl<'r, R> FromSliceShouldBeOk<'r> for R
+where
+    R: Reader<'r>,
+{
+    fn from_slice_should_be_ok(slice: &'r [u8]) -> Self {
+        match Self::from_slice(slice) {
+            Ok(ret) => ret,
+            Err(err) => {
+                use sentry::{capture_message, with_scope, Level};
+                with_scope(
+                    |scope| scope.set_fingerprint(Some(&["ckb-types", "serialization"])),
+                    || {
+                        capture_message(
+                            &format!(
+                                "failed to convert from slice: \
+                                 reason: {}; \
+                                 data: 0x{}.",
+                                err,
+                                hex_string(slice).unwrap()
+                            ),
+                            Level::Info,
+                        )
+                    },
+                );
+                panic!("verify slice should be ok, but {}", err);
```

Comment:
`panic!` is enough, we don't need to send the sentry message manually.

Reply:
DONE!